pkgs/servers/code-server/build-vscode-nogit.patch

```diff
@@ -0,0 +1,26 @@
+--- ./ci/build/build-vscode.sh
++++ ./ci/build/build-vscode.sh
+@@ -45,14 +45,12 @@
+   # Set the commit Code will embed into the product.json.  We need to do this
+   # since Code tries to get the commit from the `.git` directory which will fail
+   # as it is a submodule.
+-  export VSCODE_DISTRO_COMMIT
+-  VSCODE_DISTRO_COMMIT=$(git rev-parse HEAD)
++  export VSCODE_DISTRO_COMMIT=ca182b9fb51e2b1683d6e154ba5086fc7e8c3238
```

Comment:
We cannot hardcode this hear. Ideally we could just use the version attribute and do not use the hash at all.

Reply:
replaced with dummy value instead as a value is required but it doesn't seem to matter what is put in